specification/src/main/asciidoc/chapters/Sparkplug_4_Topics.adoc

```diff
@@ -269,19 +398,48 @@ Immediately upon reception of a DDEATH, any MQTT client subscribed to this devic
 data quality of all metrics to “*STALE”* and should note the time stamp when the DDEATH message was
 received.
 
-[[topics_command_dcmd]]
+[[topics_death_message_ddeath]]
+====== Topic (DDEATH)
+
+The Sparkplug topic namespace for a device Death Certificate is:
+[subs="quotes"]
+  namespace/group_id/*DDEATH*/edge_node_id/device_id
+  
+[[payloads_desc_ddeath]]
+====== Payload (DDEATH)
+
+The DDEATH message requires the following payload components.
+
+* The DDEATH must include the a seq number in the payload and it must have a value of one greater than the 
+previous MQTT message from the EoN node contained unless the previous MQTT message contained a value of 255. 
+In this case the seq number must be 0.
+
+[[command_dcmd]]
 ===== Command (DCMD)
 
 The DCMD topic provides the topic namespace used to publish metrics to any connected device. This
 means sending a new metric value to an associated metric included in the DBIRTH metric list.
+
+[[topics_command_dcmd]]
+====== Topic DCMD)
+
 [subs="quotes"]
   namespace/group_id/*DCMD*/edge_node_id/device_id
+  
+[[payloads_desc_dcmd]]
+====== Payload (DCMD)
+
+The DCMD message requires the following payload components.
+
+* The DCMD must include a timestamp denoting the DateTime the message was sent from the backend 
+application’s MQTT client.
+* The DCMD must include the metrics that need to be written to on the device.
 
 [[topics_scada_iiot_host]]
-==== SCADA / IIoT Host
+==== Host Application
```

Comment:
Since this a 'Primary Host Application' I think that should be denoted in the heading rather than just a 'Host Application'.

Reply:
OK

Reply:
Done